langchain/callbacks/tracers/base.py

```diff
@@ -45,19 +50,23 @@ def _start_trace(self, run: Run) -> None:
                     f"Parent run with UUID {run.parent_run_id} not found."
                 )
         self.run_map[str(run.id)] = run
+        self._persist_partial_run(run)
```

Comment:
Let's discuss this a bit more tomorrow. I wonder if logging to the backend on every start this way might add extra unwanted latency. I would hold off on this until we have a better solution, might be use to use a combination of `asyncio.create_task` and `asyncio.Queue` for this cc @nfcampos